settings.py

```diff
@@ -1,11 +1,14 @@
 
-import dotenv
 from unipath import Path
 from os import getenv as env
 
+import warnings
+warnings.filterwarnings('ignore', module='dotenv')
+
+import dotenv
 dotenv.load_dotenv(Path(__file__).parent.child('.env'))
 
-VERSION = '2.0.0'
+VERSION = '2.0.1'
```

Comment:
since this changes the external interface (but is backwards compatible, presumably), shouldn't this be version 2.1.0?